src/SkynetServer.Shared/Database/DatabaseContext.cs

```diff
@@ -29,8 +29,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             var session = modelBuilder.Entity<Session>();
             session.HasKey(s => new { s.AccountId, s.SessionId });
             session.HasOne(s => s.Account).WithMany(a => a.Sessions).HasForeignKey(s => s.AccountId);
+            session.Property(s => s.CreationTime).HasDefaultValueSql("NOW()");
             session.Property(s => s.SessionId).ValueGeneratedNever();
-            session.Property(s => s.ApplicationIdentifier).IsRequired();
```

Comment:
According to the specs this should still be called `ApplicationIdentifier`

Reply:
Could we change this to `AppIdentifier` on Protodesign and all implementations?

Reply:
We can do this, but all other occurrences in code would have to be changed then. But I think this should be separated from "FCM implementation" as it requires many changes across all repositories.

Reply:
Maybe it's easier then to keep `ApplicationIdentifier` in every place. If it is implemented correctly e.g. `android/de.vectordata.skynet` the key isn't longer anymore than it's value if you let the MariaDB CLI print the table.

Reply:
> If it is implemented correctly

We have to make some naming conventions on what is correct.

Nevertheless, I agree that it'd be easier to leave it as-is.

Reply:
Naming conventions are on Protodesign.